linode/manager
# upcoming: [M3-8215] - Hide monthly network transfer section for distributed regions

## Description 📝
Distributed compute instances have no transfer pool, so the Monthly Network Transfer section is irrelevant. This PR hides that section and expands the Network Transfer History chart to occupy the space.

## Changes  🔄
List any change relevant to the reviewer.
- Hide the monthly network transfer section for Linodes in a distributed region

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/1d95840b-e5e7-4df9-81d4-343b8b22ee80) | ![image](https://github.com/user-attachments/assets/8ee818dc-f890-489b-b0de-1366952111f2) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Ensure your account has the `new-dc-testing`, `edge_testing` and `edge_compute` customer tags
- Create a Linode in a distributed region if you don't already have one

### Verification steps
(How to verify changes)
- Go to a Linode's details page and then navigate to the Network tab
- The Monthly Network Transfer section should not display for Linodes in a distributed region
- The Monthly Network Transfer section should display as normal for Linodes in a core region

## As an Author I have considered 🤔

*Check all that apply*
